example_test.go

```diff
@@ -37,7 +37,11 @@ import (
 func Example() {
 	var log = controllers.Log.WithName("builder-examples")
 
-	manager, err := controllers.NewManager(controllers.GetConfigOrDie(), controllers.Options{})
+	manager, err := controllers.NewManager(controllers.GetConfigOrDie(), controllers.Options{
```

Comment:
Can you instead create a new example to show how to change leader election lease duration?